### PR TITLE
compare map and location info

### PR DESCRIPTION
**Pull request by cathydeng**
## Changes
adding stuff on the compare page above the tables. TODO: search-zip-awareness + distance from zip

## Screenshots
Screen Shot 2022-10-28 at 11 52 47 AM

## Checklist

- [ ] if adds a new page, adds accessibility tests
- [ ] if adds a new page or new interaction, adds e2e test
- [ ] if adds a new page or new interaction, sends google analytics events and updates [GA doc](https://docs.google.com/document/d/1DfQDUNZPO3B352EhUwXp0el6qrAWz8Jq9cTa1yv8Ty4/edit)
- [ ] if changes filter functionality, updates [filters doc](https://docs.google.com/document/d/1yEdo7IpHCtEKNNF6FMLapBUgcPw_8CY7wGwxKTdtJrU/edit)

Fixes https://app.asana.com/0/0/1203186341961983/f

### CODE REVIEW

File: src/pages/Search/DesktopResults.tsx [hunk 1]
```diff
@@ -39,28 +39,26 @@ function DesktopResults({ results }: { results: CareProviderSearchResult[] }) {
           key="desktop-map"
           className="position-sticky top-0"
         >
-          <div className="border-right border-left border-base-lighter">
```
Comment (cathydeng) [inline src/pages/Search/DesktopResults.tsx:42]: unrelated change here and in MobileResults - just noticed that we still had a border, which we only needed b/c of the stamen tiles that had a white bg

File: src/pages/Compare.tsx [hunk 2]
```diff
@@ -1,17 +1,33 @@
...
 export default function Compare() {
   useEffect(() => {
     logPageView();
   }, []);
 
   const [params, _setParams] = useSearchParams();
+  const location = useLocation();
+  const { t } = useTranslation();
+
+  // TODO: make this page zip-aware
```
Comment (jlhogan) [inline src/pages/Compare.tsx:25]: what does 'zip aware' mean?

**Review by jlhogan — APPROVED** [review]

File: src/pages/Compare.tsx (hunk 2 above)
Comment (cathydeng) [reply src/pages/Compare.tsx:25]: have zip from the search page as a url param and display distance from zip in the location summary
